test_unstructured_ingest/dest/azure-cognitive-search.sh

```diff
@@ -21,8 +19,8 @@ DESTINATION_INDEX="utic-test-ingest-fixtures-output-$random_id"
 # 2023-07-01-Preview, 2021-04-30-Preview, 2020-06-30-Preview
 API_VERSION=2023-07-01-Preview
 
-if [ -z "$AZURE_SEARCH_API_KEY" ]; then
-  echo "Skipping Azure Cognitive Search ingest test because AZURE_SEARCH_API_KEY env var is not set."
+if [ -z "$AZURE_SEARCH_API_KEY" ] || [ -z "$AZURE_SEARCH_ENDPOINT" ]; then
```

Comment:
Why is this changed?

Reply:
Not sure how this was passing in CI, but that endpoint was a configurable value. It broke for me locally without this change.

Reply:
https://github.com/Unstructured-IO/unstructured/blob/a8a103bc5c378900c3c17bffde26a3b41287ff89/test_unstructured_ingest/dest/azure-cognitive-search.sh#L14C1-L14C22

The variable was defined at the top (being removed here) and it is not a secret value. We've made a similar discussion in a past PR, we don't want to bloat CI values with non secret variables

Reply:
Also, the azure cognitive search credentials has recently been modified. If you have a local credential which is not updated, that might be the reason why it failed in local

Reply:
I assumed because we had that value in LastPass, it was being used in CI.

Reply:
bump - can we remove? there should never be a case where this isn't defined given L14 right?

Reply:
removed

Reply:
Agreed that it shouldn't exist in Lastpass 👍 I tried updating but having permission issues

![image](https://github.com/Unstructured-IO/unstructured/assets/39141206/c81e5d5f-72e7-4462-bc83-ff7140dc9a05)

Reply:
Updated now 👍